ingresspipe/utils/google_api_utils.py

```diff
@@ -11,7 +11,6 @@
 
 def download_creds_file():
     if not os.path.exists("./credentials.json"):
-    
         print("Retrieving Google API credentials from Synapse...")
         syn = synapseclient.Synapse()
         syn.login(SYN_UNAME, SYN_PWD, rememberMe=False)
```

Comment:
does the username and pwd not get called here?

Reply:
My apologies, I forgot to take care of this part. I will issue another PR after modifying this file. Thanks for pointing it out!

Reply:
you should be able to add your commit to this PR since it hasn't been merged yet, no need to issue another PR (unless you want to)! and np, this is the goal of code reviews :)

Reply:
I was just going over this, and it looks like this will be necessary, since, `download_creds_file()` function downloads the `credentials.json` file from its location on Synapse, and it's necessary that we ask the user to provide their username/password when downloading the file for the first time.

It's okay because this method isn't called in any of the core files, just a couple of example files which you can trigger only through the command line, and isn't directly affecting the frontend.